skopt/optimizer/optimizer.py

```diff
@@ -172,7 +173,32 @@ def _check_arguments(self, base_estimator, n_random_starts, acq_optimizer):
                 "Expected acq_optimizer to be 'lbfgs' or 'sampling', "
                 "got %s" % acq_optimizer)
 
-    def ask(self):
+    def ask(self, n_points=None, strategy="CL"):
```

Comment:
If `n_points` is set, I return a list of points, and single point if it is none. I think this might be better for users who might want to experiment with different number of points (including n_points=1), as then they do not need to account for special case when `n_points=1`.